test/integration/start_stop_delete_test.go

```diff
@@ -60,6 +60,9 @@ func TestStartStop(t *testing.T) {
 			"--extra-config",
 			"kubeadm.ignore-preflight-errors=SystemVerification",
 		}},
+		{"registry_mirror", []string{
+			"--registry-mirror=https://localhost",
```

Comment:
Thank you so much for updating the PR and doing the improvements. by testing I meant unit testing. something similar to this, 
https://github.com/kubernetes/minikube/blob/master/cmd/minikube/cmd/config/config_test.go 

but that can be done in a separate PR, for now I suggest removing this option from the integration test.

Reply:
Sounds good. I have removed it and will add the test in a separate PR